inst/plumber/v1/plumber.R

```diff
@@ -263,3 +262,58 @@ function(req) {
       povline = params$povline,
       lkup = lkups)
 }
+
+# UI Endpoints: Country Profiles ----------------------------------------
+
+#* Return Key Indicator 1 (national poverty)
+#* @get /api/v1/cp-key-indicator-1
+#* @serializer json
+function(req) {
+  lkups$cp_lkups$key_indicators$ki1
+}
+
+#* Return Key Indicator 2 (headcount)
+#* @get /api/v1/cp-key-indicator-2
+#* @param country:[chr] country iso3 code
+#* @param povline:[dbl] Poverty Line
+#* @serializer json
+function(req) {
+  params <- req$argsQuery
+  params$lkup <- lkups
+  do.call(pipapi::ui_cp_ki_headcount, params)
+}
```

Comment:
This one returns an error when called without parameters. Add meaningful defaults or informative error message.